inspire_mitmproxy/management_service.py

```diff
@@ -22,11 +22,101 @@
 
 """Service used to orchestrate fake services."""
 
-from typing import List
+from autosemver import get_current_version
+from json import dumps as json_dumps, loads as json_loads, JSONDecodeError
+from mitmproxy.net.http.status_codes import RESPONSES
+from os import environ, getcwd, listdir
+from os.path import isdir, isfile, join as path_join
+from typing import cast, Dict, List, Optional, Union
+from urllib.parse import urlparse
 
 from .base_service import BaseService
+from .errors import RequestNotHandledInService, InvalidRequest
 
 
 class ManagementService(BaseService):
+    SERVICE_HOSTS = ['mitm-manager.local']
+
     def __init__(self, services: List[BaseService]) -> None:
         self.services = services
+        self.config = {
+            'active_scenario': None,
+        }
+
+    def process_request(self, request: dict) -> dict:
+        parsed_url = urlparse(request['uri'])
+        path = parsed_url.path
+        method = request['method']
+
+        if path == '/services' and method == 'GET':
+            return self.build_response(200, self.get_services())
+        elif path == '/scenarios' and method == 'GET':
+            return self.build_response(200, self.get_scenarios())
+        elif path == '/config' and method == 'GET':
+            return self.build_response(200, self.get_config())
+        elif path == '/config' and method == 'PUT':
+            return self.build_response(204, self.put_config(request))
+        elif path == '/config' and method == 'POST':
+            return self.build_response(201, self.post_config(request))
+
+        raise RequestNotHandledInService(self, request)
+
+    def get_services(self) -> dict:
+        return {
+            idx: {
+                'class': type(service).__name__,
+                'service_hosts': service.SERVICE_HOSTS,
+            }
+            for idx, service in enumerate([cast(BaseService, self)] + self.services)
+        }
+
+    def get_scenarios(self) -> dict:
+        response: Dict[str, Dict[str, List[str]]] = {}
+        path = environ.get('SCENARIOS_PATH', './scenarios/')
```

Comment:
Add a jira ticket to tackle configuration (maybe using ConfigParser from stdlib)